FinniversKit/Sources/Components/Consent/BannerTransparency/BannerTransparencyViewModel.swift

```diff
@@ -28,6 +30,13 @@ extension BannerTransparencyViewModel {
     }
 }
 
+// MARK: - LogoType Enum
+
+public enum LogoType {
```

Comment:
The FinniversKit components should in general not care about what specific type of images is being used. Could this be a more generic ID of some kind that is configured on setup?

Reply:
Removed this part of the PR - thank you again for your input Björn :D